diff --git a/polkadot/runtime/rococo/src/governance/mod.rs b/polkadot/runtime/rococo/src/governance/mod.rs
index ef2adf60753..218c0a3c837 100644
--- a/polkadot/runtime/rococo/src/governance/mod.rs
+++ b/polkadot/runtime/rococo/src/governance/mod.rs
@@ -35,7 +35,7 @@ mod fellowship;
 pub use fellowship::{FellowshipCollectiveInstance, FellowshipReferendaInstance};
 
 parameter_types! {
-	pub const VoteLockingPeriod: BlockNumber = 7 * DAYS;
+	pub const VoteLockingPeriod: BlockNumber = 1;
 }
 
 impl pallet_conviction_voting::Config for Runtime {
diff --git a/polkadot/runtime/rococo/src/governance/tracks.rs b/polkadot/runtime/rococo/src/governance/tracks.rs
index 3765569f183..ed226f4ef65 100644
--- a/polkadot/runtime/rococo/src/governance/tracks.rs
+++ b/polkadot/runtime/rococo/src/governance/tracks.rs
@@ -212,10 +212,10 @@ const TRACKS_DATA: [(u16, pallet_referenda::TrackInfo<Balance, BlockNumber>); 15
 			name: "small_tipper",
 			max_deciding: 200,
 			decision_deposit: 1 * 3 * CENTS,
-			prepare_period: 1 * MINUTES,
-			decision_period: 14 * MINUTES,
-			confirm_period: 4 * MINUTES,
-			min_enactment_period: 1 * MINUTES,
+			prepare_period: 1,
+			decision_period: 1,
+			confirm_period: 1,
+			min_enactment_period: 1,
 			min_approval: APP_SMALL_TIPPER,
 			min_support: SUP_SMALL_TIPPER,
 		},
@@ -226,10 +226,10 @@ const TRACKS_DATA: [(u16, pallet_referenda::TrackInfo<Balance, BlockNumber>); 15
 			name: "big_tipper",
 			max_deciding: 100,
 			decision_deposit: 10 * 3 * CENTS,
-			prepare_period: 4 * MINUTES,
-			decision_period: 14 * MINUTES,
-			confirm_period: 12 * MINUTES,
-			min_enactment_period: 3 * MINUTES,
+			prepare_period: 1,
+			decision_period: 1,
+			confirm_period: 1,
+			min_enactment_period: 1,
 			min_approval: APP_BIG_TIPPER,
 			min_support: SUP_BIG_TIPPER,
 		},
diff --git a/polkadot/runtime/rococo/src/lib.rs b/polkadot/runtime/rococo/src/lib.rs
index 03e96ab388b..f4a58d1792c 100644
--- a/polkadot/runtime/rococo/src/lib.rs
+++ b/polkadot/runtime/rococo/src/lib.rs
@@ -451,7 +451,7 @@ parameter_types! {
 	pub const ProposalBond: Permill = Permill::from_percent(5);
 	pub const ProposalBondMinimum: Balance = 2000 * CENTS;
 	pub const ProposalBondMaximum: Balance = 1 * GRAND;
-	pub const SpendPeriod: BlockNumber = 6 * DAYS;
+	pub const SpendPeriod: BlockNumber = 1;
 	pub const Burn: Permill = Permill::from_perthousand(2);
 	pub const TreasuryPalletId: PalletId = PalletId(*b"py/trsry");
 	pub const PayoutSpendPeriod: BlockNumber = 30 * DAYS;
diff --git a/polkadot/runtime/westend/src/governance/mod.rs b/polkadot/runtime/westend/src/governance/mod.rs
index d027f788d71..de6f36ecfce 100644
--- a/polkadot/runtime/westend/src/governance/mod.rs
+++ b/polkadot/runtime/westend/src/governance/mod.rs
@@ -32,7 +32,7 @@ mod tracks;
 pub use tracks::TracksInfo;
 
 parameter_types! {
-	pub const VoteLockingPeriod: BlockNumber = 7 * DAYS;
+	pub const VoteLockingPeriod: BlockNumber = 1;
 }
 
 impl pallet_conviction_voting::Config for Runtime {
diff --git a/polkadot/runtime/westend/src/governance/tracks.rs b/polkadot/runtime/westend/src/governance/tracks.rs
index 3765569f183..ed226f4ef65 100644
--- a/polkadot/runtime/westend/src/governance/tracks.rs
+++ b/polkadot/runtime/westend/src/governance/tracks.rs
@@ -212,10 +212,10 @@ const TRACKS_DATA: [(u16, pallet_referenda::TrackInfo<Balance, BlockNumber>); 15
 			name: "small_tipper",
 			max_deciding: 200,
 			decision_deposit: 1 * 3 * CENTS,
-			prepare_period: 1 * MINUTES,
-			decision_period: 14 * MINUTES,
-			confirm_period: 4 * MINUTES,
-			min_enactment_period: 1 * MINUTES,
+			prepare_period: 1,
+			decision_period: 1,
+			confirm_period: 1,
+			min_enactment_period: 1,
 			min_approval: APP_SMALL_TIPPER,
 			min_support: SUP_SMALL_TIPPER,
 		},
@@ -226,10 +226,10 @@ const TRACKS_DATA: [(u16, pallet_referenda::TrackInfo<Balance, BlockNumber>); 15
 			name: "big_tipper",
 			max_deciding: 100,
 			decision_deposit: 10 * 3 * CENTS,
-			prepare_period: 4 * MINUTES,
-			decision_period: 14 * MINUTES,
-			confirm_period: 12 * MINUTES,
-			min_enactment_period: 3 * MINUTES,
+			prepare_period: 1,
+			decision_period: 1,
+			confirm_period: 1,
+			min_enactment_period: 1,
 			min_approval: APP_BIG_TIPPER,
 			min_support: SUP_BIG_TIPPER,
 		},
diff --git a/polkadot/runtime/westend/src/lib.rs b/polkadot/runtime/westend/src/lib.rs
index bbb010f60bf..ab0a6569482 100644
--- a/polkadot/runtime/westend/src/lib.rs
+++ b/polkadot/runtime/westend/src/lib.rs
@@ -711,7 +711,7 @@ parameter_types! {
 	pub const ProposalBond: Permill = Permill::from_percent(5);
 	pub const ProposalBondMinimum: Balance = 2000 * CENTS;
 	pub const ProposalBondMaximum: Balance = 1 * GRAND;
-	pub const SpendPeriod: BlockNumber = 6 * DAYS;
+	pub const SpendPeriod: BlockNumber = 1;
 	pub const Burn: Permill = Permill::from_perthousand(2);
 	pub const TreasuryPalletId: PalletId = PalletId(*b"py/trsry");
 	pub const PayoutSpendPeriod: BlockNumber = 30 * DAYS;
